main.go

```diff
@@ -281,6 +284,7 @@ type FileConfig struct {
 
 var check = flag.Bool("check", false, "Validate the config file and then exit")
 var cfg = flag.String("config", "config.yml", "Path to a config file")
+var emailsCfg = flag.String("emails-config", "emails_config.yml", "Path to a emails config file")
```

Comment:
Could we preserve backwards compatibility in theory by defaulting this to the same file as cfg? Or does the file structure change?